src/Grids/vertically_stretched_rectilinear_grid.jl

```diff
@@ -206,3 +206,14 @@ Adapt.adapt_structure(to, grid::VerticallyStretchedRectilinearGrid{FT, TX, TY, T
 
 @inline znode(::Type{Center}, k, grid::VerticallyStretchedRectilinearGrid) = @inbounds grid.zᵃᵃᶜ[k]
 @inline znode(::Type{Face}, k, grid::VerticallyStretchedRectilinearGrid) = @inbounds grid.zᵃᵃᶠ[k]
+
+
+
+#
+# Get minima of grid
+#
+
+min_Δx(grid::VerticallyStretchedRectilinearGrid) = grid.Δx
+min_Δy(grid::VerticallyStretchedRectilinearGrid) = grid.Δy
+min_Δz(grid::VerticallyStretchedRectilinearGrid) = minimum(grid.Δzᵃᵃᶜ[1:grid.Nz])
```

Comment:
You might need

```julia
min_Δz(grid::VerticallyStretchedRectilinearGrid) = minimum(view(grid.Δzᵃᵃᶜ, 1:grid.Nz))

Reply:
Hmm, because of the GPU?

Reply:
You also might be able to use

```julia
min_Δz(grid::VerticallyStretchedRectilinearGrid) = minimum(znodes(Center, grid))
```

Reply:
Yes, I think you even want this on the CPU because `grid.Δzᵃᵃᶜ[1:grid.Nz]` allocates memory. We'll see if tests pass.